tests/debuginfo/range-types.rs

```diff
@@ -1,7 +1,10 @@
 // Testing the display of range types in cdb.
 
-// cdb-only
-//@ min-cdb-version: 10.0.18317.1001
+//@ only-cdb
```

Comment:
Remark: `// cdb-only` had no effect, but this test only contained `cdb` so this really doesn't change anything.